Specs/Display.lean

```diff
@@ -1,30 +1,35 @@
+import Specs.Config
+import Specs.Pretty
+
 /-! Data structure for representing the execution of a test suite. -/
 
 namespace Specs.Display
 
 inductive TestTree where
-  | group (name: String) (tests: Array TestTree)
-  | test (succeded: Bool) (message: String) (explanation: Option String)
+  | group (config : Config) (name: String) (tests: Array TestTree)
```

Comment:
It smells like bad code :S I think that we can just pass Config as part of a `render` function or just wrap the TestTree with something like `testTree.with(config).toString`